### ISSUE TITLE
bugs/features suggestions for CEMRG WaThCa perspective

### ISSUE THREAD

**Issue by dariadg** [CemrgAppDevelopers/CemrgApp#17]
The two that happened throughout the year.

Everything in WaThCa perspective:

   1. After doing a couple of cases, the App would quit.
	    It would always quit after I’ve loaded a new image, then pressed: “Segment Images”. 
            It would ask: “Do you have a segmentation to load”. I’d answer: “No”. 
            It shuts down completely. 
   2. Not an issue but a request:
            After you found centrelines
	    And displayed clippers
	    If you want to change your centrelines, you can’t revert back to find centrelines
   Would be great, as can’t always notice if the centreline is going  the wrong way. If you want to do it again, have to do 
   PV clipper again.
The rest has been good.

ALSO: There are a lot of other button combinations that I sometimes accidentally pressed and some cool things happen (image attached) but don’t know what they are. May be extend the list of explained commands?
[
<img width="571" alt="Screenshot 2020-07-13 at 19 57 01" src="https://user-images.githubusercontent.com/42388842/87338329-aa632780-c54d-11ea-9242-3d82f53185d1.png">
](url)

**Comment by alonsoJASL** [comment]
Which platform is this error happening on?

**Comment by dariadg** [comment]
On both, older and current versions.

**Comment by alonsoJASL** [comment]
Thanks. As this has multiple issues, I'll split it into multiple issues, so we can work on them in parallel and prioritise them all properly.

**Comment by alonsoJASL** [comment]
New issues #18 #19 and #20 have been created and prioritised. We will follow through in those. Thanks @dariadg for letting us know.
